### DISCUSSION TITLE
which course?

### DISCUSSION THREAD

**Discussion by itsmohammadh** [Cyfrin/foundry-full-course-cu#561] [Q&A]
Hi guys, i want to started with patrick course but i dont know which one!
i want to learn completely solidity and become a blockchain developer but i confused( which better for beginner ) , do you think it is better to see javaScript course or Foundry?
Thanks for the advice.

**Comment by shikhar229169** [comment]
I would recommend you to start with foundry course because I think it would be easier for a beginner to start with and also its the latest one.
Also we have a bunch of new people here, so the community support is very strong and you can collaborate with new learners over here.

And also sometimes working with javascript is a headache because of the package dependencies issue. A lot of people are having difficulty due to that. Because the js course was based on the version which was the latest at that time and the current new version of the package has changed a number of functions and also some functions are removed.

So, my recommendation is to start this Foundry course.

**Reply by itsmohammadh** [reply]
thank you for comment @shikhar229169 🙏
